anndata/compat/__init__.py

```diff
@@ -26,6 +26,16 @@ def __repr__():
             return "mock zarr.core.Array"
 
 
+try:
```

Comment:
So in line 14
from .._core.index import _subset

makes module loading fails due to a circular import in anndata._core.index (which uses `AwkArray` initialized here).
I think I have to move the mock import of AwkArray somewhere else, not in this __init__.py file. shall I just make a new mdoule in compat where all the compatibility imports arestored?
@ivirshup

Reply:
Yup, sounds good